ansible/roles/nginx/tasks/certbot.yml

```diff
@@ -0,0 +1,28 @@
+- name: Install certbot
+  pip:
+    name:
+      - certbot
+      - certbot-nginx
+    executable: pip3
+
+- name: Register certbot
```

Comment:
Yes, when Ansible executes the very first time it will create the `.registered` file, and for the next executions it will skip it

- first execution
```
TASK [nginx : Register certbot] ******************
changed: [test-nginx-0]
```
- next execution
```
TASK [nginx : Register certbot] *******************
ok: [test-nginx-0]
```